src/components/FormInput.tsx

```diff
@@ -1,29 +1,48 @@
 import React, { useState } from 'react';
+import TodoListProps from '../TypescriptProps';
 
 interface FormInputProp {
     addTask: any;
+    listOfItems: TodoListProps[];
 }
 
-const FormInput: React.FC<FormInputProp> = ({ addTask }) => {
-
+const FormInput: React.FC<FormInputProp> = ({ addTask, listOfItems }) => {
     const [ userInput, setUserInput ] = useState('');
-
+    const [ inputError, setInputError ] = useState('');
+    let listCheck = listOfItems.map(listDes => listDes.description);
+    const errorCheck = () => {
+        if (!userInput) {
+            setInputError("No blank spaces");
+            return false;
+        }
+        if (listCheck.includes(userInput.trim())) {
```

Comment:
probably need to trim the list before you store it as state

Reply:
Updated and whitespace is now trimming as expected.